src/preview/Preview.tsx

```diff
@@ -476,7 +476,7 @@ const SlidePreview = ({
     >
       {masterSlide?.objects?.map((o, i) => (
         <SlideObjectPreview
-          key={i}
```

Comment:
What does this line do?

Reply:
My bad! I was just testing something (just push the actual fix)